premerge/post-submit-testing.md

```diff
@@ -0,0 +1,114 @@
+# Post Submit Testing
+
+## Introduction
+
+While this infrastructure is focused on premerge testing, it is also important
+to make sure that the specific configuration we are testing is tested post
+commit as well. This document outlines the motivation for the need to test this
```

Comment:
Explain what you mean by 'tested post commit".

Reply:
Added a paragraph to help explain the flow here.